### ISSUE TITLE
Error on rendering ungrouped StaticList

### ISSUE THREAD

**Issue by toaothink** [telerik/kendo-ui-core#784]
There seems to be an Issue on this line https://github.com/telerik/kendo-ui-core/blob/master/src/kendo.list.js#L1827 under certain cirumstances. In my case i'm rendering a DropDownList that's connected to a remote DataSource and there's an error thrown on this line because this.dataSource.group() returns an empty object instead of an array. The DataSource loaded the remote data correctly, there just seems to be an error in the grouping functionality which I do not even use.

I created a post on StackOverflow that contains more information about my setup: http://stackoverflow.com/questions/29769776/error-rendering-kendo-ui-dropdownlist-bound-to-remote-datasource

**Comment by ggkrustev** [comment]
Hi @toaothink,

For some reason, the group value is not normalized. Would it be possible to create a repro demo using [Kendo Dojo](http://dojo.telerik.com/)? More guidelines can be found [here](https://github.com/telerik/kendo-ui-core/blob/master/CONTRIBUTING.md#reporting-bugs).

**Comment by toaothink** [comment]
Hi @ggkrustev 

Thank you for getting in touch. I am working on an example in the Kendo Dojo. I tried to start as simple as possible and so far I could not reproduce the issue. In my real world app the DropDownList is embeded in a view that get's rendered inside a kendoWindow and I'm using external Templates to define the views. The data is consumed from a WCF Service that is hosted inside a MS SharePoint Web App. So there are a lots of possible sources for this error to rule out and I'm strugging to pick the right order of further abstractions to implement in the Kendo Dojo. Since the code used to work until I had to use a local version of the framwork instead of the CDN version Telerik provides I guess I'll try to figure out the differences between the two versions of my code and the framework before I invert more time into reproducing the issue in the [Dojo](http://dojo.telerik.com/aGefI).
